examples/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ function(cutlass_example_add_executable NAME)
   if (NOT CUTLASS_ENABLE_SYCL)
     SET(ADD_CUDA ON)
   endif()
+  
+  if (DPCPP_SYCL_ARCH STREQUAL "sm_90a")
+    SET(ADD_CUDA ON)
+  endif()
```

Comment:
For context: this is needed to call the function that initialise the TMA descriptor